src/main/java/no/uib/inf112/core/testutils/HeadlessMapHandler.java

```diff
@@ -13,17 +13,17 @@ public HeadlessMapHandler(String map) {
 
     @Override
     public void render(@NotNull Batch batch) {
-
+        //Do nothing
     }
```

Comment:
More descriptive to write "skeleton code:  made for running in test environment nowhere to render too."?